src/main/java/com/boot/cart/controller/CartController.java

```diff
@@ -32,7 +32,7 @@ public class CartController {
     @Autowired
```

Comment:
is it working without a @RequestMapping("/cart") preceding CartController declaration?

Reply:
yes, the path for each specific microservice is set up in the application.properties from the zuul-servic
![Capture2](https://user-images.githubusercontent.com/59165649/169452250-33f846ab-beab-4f00-9588-4274b59c7767.PNG)
e